dev/io.openliberty.data.internal.persistence/resources/io/openliberty/data/internal/persistence/resources/CWWKDMessages.nlsprops

```diff
@@ -1073,3 +1073,23 @@ CWWKD1105.keyword.in.orderby.explanation=Entity attribute names that contain \
  follow the Jakarta Data Query by Method Name pattern.
 CWWKD1105.keyword.in.orderby.useraction=Rename the entity attribute or switch \
  to the OrderBy annotation to specify the ordering.
+
+CWWKD1106.init.timed.out=CWWKD1106E: The {0} repository that requires the \
+ {1} dataStore and is provided or used by the {2} application artifact did not \
+ become available within {3} seconds. Confirm that the {2} application artifact \
+ starts and that the {1} dataStore is configured correctly and is accessible to \
+ the application artifact at the point when the {0} repository is first accessed.
+CWWKD1106.init.timed.out.explanation=A repository did not become available \
+ within the allotted amount of time.
+CWWKD1106.init.timed.out.useraction=Check the configuration of the dataStore \
+ and that is it accessible to application artifacts that define and use the \
+ repository. Also confirm that the application and all of its modules start \
+ successfully.
+
+CWWKD1107.init.timed.out.checkpoint=CWWKD1107E: The {0} application artifact \
+ accessed the {1} repository before a checkpoint of the application completed. \
+ Update the {0} application to avoid using the repository before a checkpoint.
+CWWKD1107.init.timed.out.checkpoint.explanation=Repositories are not available \
```

Comment:
The explanation "Repositories are not available to be used by applications before completion of a checkpoint." is okay as is. To simplify the wording, you might change it to:

Applications cannot use repositories before completion of a checkpoint.